Cake.Recipe/Content/analyzing.cake

```diff
@@ -38,16 +49,27 @@ BuildParameters.Tasks.DupFinderTask = Task("DupFinder")
 );
 
 BuildParameters.Tasks.InspectCodeTask = Task("InspectCode")
-    .WithCriteria(() => BuildParameters.BuildAgentOperatingSystem == PlatformFamily.Windows, "Skipping due to not running on Windows")
     .WithCriteria(() => BuildParameters.ShouldRunInspectCode, "Skipping because InspectCode has been disabled")
-    .Does<BuildData>(data => RequireTool(ToolSettings.ReSharperTools, () => {
+    .Does<BuildData>(data => RequireTool(BuildParameters.IsDotNetCoreBuild ? ToolSettings.ReSharperGlobalTools : ToolSettings.ReSharperTools, () => {
         var inspectCodeLogFilePath = BuildParameters.Paths.Directories.InspectCodeTestResults.CombineWithFilePath("inspectcode.xml");
 
         var settings = new InspectCodeSettings() {
             SolutionWideAnalysis = true,
             OutputFile = inspectCodeLogFilePath
         };
 
+        // Workaround until 1.0.0+ version of cake is released
+        if (BuildParameters.IsDotNetCoreBuild && BuildParameters.BuildAgentOperatingSystem == PlatformFamily.Windows)
+        {
+            settings.ToolPath = Context.Tools.Resolve("jb.exe");
+            settings.ArgumentCustomization = args => args.Prepend("inspectcode");
+        }
+        else if(BuildParameters.IsDotNetCoreBuild)
+        {
+            settings.ToolPath = Context.Tools.Resolve("jb");
+            settings.ArgumentCustomization = args => args.Prepend("inspectcode");
+        }
+
```

Comment:
We'd probably want an else if statement here for when the user isn't running on windows, to warn the user and not continuing with the task.

Or we could update the `WithCriteria` function you removed: ie
```
WithCriteria(() => BuildParameters.IsDotNetCoreBuild || BuildParameters.BuildAgentOperatingSystem == PlatformFamily.Windows
```

Reply:
:+1: